etc/config-template.toml

```diff
@@ -134,6 +134,45 @@
 ## When the pending write bytes exceeds this threshold, the "scheduler too busy" error is displayed.
 # scheduler-pending-write-threshold = "100MB"
 
+[storage.block-cache]
+## Whether to create a shared block cache for all RocksDB column families.
+##
+## Block cache is used by RocksDB to cache uncompressed blocks. Big block cache can speed up read.
+## It is recommended to turn on shared block cache. Since only the total cache size need to be
+## set, it is easier to config. In most cases it should be able to auto-balance cache usage
+## between column families with standard LRU algorithm.
+##
+## The rest of config in the storage.block-cache session is effective only when shared block cache
+## is on.
+# shared = true
+
+## Size of the shared block cache. Normally it should be tuned to 30%-50% of system's total memory.
+## When the config is not set, it is decided by the sum of the following fields or their default
+## value:
+##   * rocksdb.defaultcf.block-cache-size or 25% of system's total memory
+##   * rocksdb.writecf.block-cache-size   or 15% of system's total memory
+##   * rocksdb.lockcf.block-cache-size    or  2% of system's total memory
+##   * raftdb.defaultcf.block-cache-size  or  2% of system's total memory
+##
+## To deploy multiple TiKV nodes on a single physical machine, configure this parameter explicitly.
+## Otherwise, the OOM problem might occur in TiKV.
+# capacity = "1GB"
+
+## Shard the block cache into 2^(num-shard-bits) shards. More shards reduce lock contention for
+## accessing the block cache on read, but increae cache fragmentation. Set the value to -1 to
+## allow RocksDB auto-config number of shards by cache size.
+# num-shard-bits = -1
+
+## Whether to fail read when the cache reaches its capacity, but no cache items can be evicted.
+# strict-capacity-limit = false
+
+## Config ratio of cache capacity allocate to the high-pri pool. Should be a value between 0.0-1.0.
+## When set to value greater than zero, mid-point insertion strategy is enabled. Cache items will
+## first inserted to the middle of LRU list at the position decided by the high-pri pool ratio,
+## and move to the tail of LRU list after at least one cache hit. Enabling mid-point insertion
+## will provide some resistence to large table scan evicting useful cache items.
+# high-pri-pool-ratio = 0.0
```

Comment:
maybe we shouldn't even mention `num-shard-bits`, `strict-capacity-limit` and `high-pri-pool-ratio` in the template?

Reply:
Good idea, less is more.